src/components/views/adminView/Cities.jsx

```diff
@@ -0,0 +1,97 @@
+import React, { useState } from 'react'
+import { TextField, Input, CircularProgress } from '@material-ui/core'
+import { fetchData } from '../../../utils'
+import CheckCircleIcon from '@material-ui/icons/CheckCircle'
+
+import Grid from '@material-ui/core/Grid'
+import Button from '@material-ui/core/Button'
+
+export const Cities = ({ useStyles }) => {
+  const [city, setCity] = useState([])
+  const [adding, setAdding] = useState(false)
+  const [added, setAdded] = useState(false)
+  const classes = useStyles()
+
+  const handlerCityChange = (e) => {
+    setCity(e.target.value)
+  }
+  const addNext = () => {
```

Comment:
i would recomend to change this function name to something like 'goNext' etc.. 

AddNext kojarzy się bardziej z jakaś akcją, dodaniem czegoś